src/Uno.UI/DataBinding/BindingPath.BindingItem.cs

```diff
@@ -478,10 +488,63 @@ private IDisposable SubscribeToPropertyChanged()
 
 			public void Dispose()
 			{
-				_disposed = true;
+				IsDisposed = true;
 				_propertyChanged.Dispose();
 			}
 
+			private bool IsDisposed
+			{
+				get => _flags.HasFlag(Flags.Disposed);
```

Comment:
I'm not sure how often all these `HasFlag` calls happen, but they might box unnecessarily. Doing a bitwise AND will be more efficient.

Reply:
Those flags are used very often, I'll make the change.